tests/test_s1_l0.py

```diff
@@ -371,18 +375,20 @@ def test_get_adgs_catalog_data(endpoint, status):
             status=status,
         )
 
+    rs_client = AuxipClient(endpoint, "", username, [])
+    adgs_res = get_adgs_catalog_data.fn(rs_client, collection, files_list)
+
     if "bad_endpoint" not in endpoint:
-        adgs_res = get_adgs_catalog_data.fn(endpoint, username, collection, files_list, apikey)
         print(adgs_res)
         if int(status) == 200:
             assert adgs_res == adgs_catalog
         else:
             assert adgs_res is None
     else:
-        assert get_adgs_catalog_data.fn(endpoint, username, collection, files_list, apikey) is None
+        assert adgs_res is None
 
 
-# TODO: The unit testing for this prefect flow does not work
+# TODO: The unit testing for this prefect flow does not work (?)
```

Comment:
@agrosu1978 you need to fix that test before merging to develop ?

Reply:
No, I will remove it. All the functions used in the flow are tested with the others tests.